dev/io.openliberty.netty.internal.tls.impl/src/io/openliberty/netty/internal/tls/impl/NettyTlsProviderImpl.java

```diff
@@ -0,0 +1,326 @@
+ * String, String, String, Boolean, SSLConnectionLink)}
+ *
+ */
+@Component(configurationPid = "io.openliberty.netty.internal.tls", immediate = true, service = NettyTlsProvider.class, property = { "service.vendor=IBM" })
+public class NettyTlsProviderImpl implements NettyTlsProvider {
+
+    private static final TraceComponent tc = Tr.register(NettyTlsProviderImpl.class);
+
+    static final String ALIAS_KEY = "alias";
+    
+    /**
+     * DS deactivate
+     * 
+     * @param ctx
+     * @param reason
+     */
+    @Activate
+    protected void activate(ComponentContext ctx){
+        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            Tr.debug(tc, "NettyTlsProviderImpl activate " + this);
+        }
+    }
+    
+    /**
+     * DS deactivate
+     * 
+     * @param ctx
+     * @param reason
+     */
+    @Modified
+    protected void modified(ComponentContext ctx){
+        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            Tr.debug(tc, "NettyTlsProviderImpl modified " + this);
+        }
+    }
+
+
+    /**
+     * DS deactivate
+     * 
+     * @param ctx
+     * @param reason
+     */
+    @Deactivate
+    protected void deactivate(ComponentContext ctx, int reason){
+        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            Tr.debug(tc, "NettyTlsProviderImpl deactivate " + this);
+        }
+    }
+
+    /**
+     * Build a {@link io.netty.handler.ssl.SslContext} for an outbound connection
+     * 
+     * @param Map<String, Object> sslOptions
+     * @param String host
+     * @param String port
+     * @return SslContext
+     */
+    public SslContext getOutboundSSLContext(Map<String, Object> sslOptions, String host,
+            String port) {
+
+        SSLContext jdkContext;
+        try {
+        	Properties props = createProps(sslOptions);
+            String alias = (String)props.getProperty(ALIAS_KEY);
+            jdkContext = getSSLContext(alias, props, false, host, port, port, null);
+            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(tc, "getOutboundSSLContext SSLContext:", jdkContext);
+            }
+        } catch (Exception e) {
+            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(tc, "getOutboundSSLContext exception caught creating SSLContext: " + e);
+            }
+            return null;
+        }
+
+        try {
+            SslContext nettyContext = new JdkSslContext(jdkContext, true, null, SupportedCipherSuiteFilter.INSTANCE,
+                    null, ClientAuth.OPTIONAL, null, false);
+            return nettyContext;
+        } catch (Exception e) {
+            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(tc, "getOutboundSSLContext exception caught creating JdkSslContext: " + e);
```

Comment:
I think this should be traced higher than debug and at a level that would appear in a standard customer trace file that would come in if the error occurred